### PR TITLE
Updated to work with latest IBM DFDL Cross Tester and plugin.

### PR DESCRIPTION
**Pull request by mbeckerle**

### CODE REVIEW

**Comment by mbeckerle** [comment]
Oops. Not trying to merge onto master.

**Comment by mbeckerle** [comment]
I guess you have to show it as merge onto master. The intention here is to stay on a branch until the release of daffodil that supports this functionality is public.

File: 2013/build.sbt [hunk 1]
```diff
@@ -1,34 +1,36 @@
...
+    inThisBuild(List(
+      organization := "com.tresys",
+      version      := "0.1.0-SNAPSHOT",
+      scalaVersion := "2.12.6",
+      crossPaths := false,
+      testOptions += Tests.Argument(TestFrameworks.JUnit, "-v"),
+    )),
+    name := "dfdl-nacha",
+    libraryDependencies := Seq(
+      "org.apache.daffodil" %% "daffodil-tdml-processor" % "latest.integration" % "test",
```
Comment (mbeckerle) [inline 2013/build.sbt]: This "latest.integration" is useful for snapshots. But this build.sbt should work fine with "2.2.0" here. Only the cross-tester will require the 2.3.0 release of daffodil, or a development snapshot.

**Review by stevedlawrence — COMMENTED** [review]
+1 :+1:

**Comment by mbeckerle** [comment]
Closing. Obsolete.

**Pull request closed without merging**